config.js

```diff
@@ -1,3 +1,5 @@
-// process.env.API_URL = 'http://localhost:9876';
-process.env.API_URL = 'https://dev.fiftynifty.org/api';
+
```

Comment:
I'll add config.js to .gitignore
no need to push local changes to it with each commit.

In the meantime, can you remove this file from the pull request?